README.md

```diff
@@ -55,17 +52,11 @@ To stop the container, first try CTRL+C. If that does not work, run `docker ps |
 ### Setup
 
 * Run Selenium Server
-* Run the Firefox Content Server locally: `npm start`
-* Run an instance of the [fxa-auth-server](https://github.com/mozilla/fxa-auth-server) at 127.0.0.1:9000.
 
 e.g. in shell form:
 
 ```sh
-java -jar selenium-server-standalone-2.43.1.jar &
-cd fxa-auth-server
-npm start &
-cd ../fxa-content-server
-npm start &
+java -jar selenium-server-standalone-2.43.1.jar 
```

Comment:
@vladikoff What about just using some npm module which includes selenium? Maybe something like https://github.com/JonaMX/selenium-standalone-jar

Not sure if that'd simplify the setup any, or just make the install slower and 30MB bigger.